omniscidb/Shared/quantile.h

```diff
@@ -0,0 +1,343 @@
+/*
+ * Copyright (C) 2023 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include "InlineNullValues.h"
+#include "SimpleAllocator.h"
+
+#include "IR/OpTypeEnums.h"
+
+#include <algorithm>
+#include <cmath>
+#include <cstdint>
+#include <vector>
+
+namespace hdk::quantile {
+
+namespace details {
+
+class ChunkedArray {
+ public:
+  struct Chunk {
+    int8_t* data;
+    // Size holds number of elements, not bytes. Element size
+    // is determined on push.
+    size_t size;
```

Comment:
Why not just call it that instead of having a comment?

Reply:
Indeed. Renamed to `max_elems`.